core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/udf/python/PythonTableReducerOpDesc.scala

```diff
@@ -0,0 +1,57 @@
+package edu.uci.ics.texera.workflow.operators.udf.python
+
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import edu.uci.ics.texera.workflow.common.metadata.{
+  InputPort,
+  OperatorGroupConstants,
+  OperatorInfo,
+  OutputPort
+}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{OperatorSchemaInfo, Schema}
+
+class PythonTableReducerOpDesc extends PythonOperatorDescriptor {
+  @JsonSchemaTitle("Output columns")
+  var lambdaAttributeUnits: List[LambdaAttributeUnit] = List()
+  override def numWorkers(): Int = 1
+
+  override def getOutputSchema(schemas: Array[Schema]): Schema = {
+    if (lambdaAttributeUnits.isEmpty) {
+      schemas(0)
+    } else {
+      val outputSchemaBuilder = Schema.newBuilder
+      for (unit <- lambdaAttributeUnits) {
+        outputSchemaBuilder.add(unit.attributeName, unit.attributeType)
+      }
+      outputSchemaBuilder.build()
+    }
+  }
+
+  override def operatorInfo: OperatorInfo =
+    OperatorInfo(
+      "Python Table Reducer",
+      "Reduce Table to Tuple",
+      OperatorGroupConstants.UDF_GROUP,
+      inputPorts = List(InputPort()),
+      outputPorts = List(OutputPort())
+    )
+
+  override def generatePythonCode(operatorSchemaInfo: OperatorSchemaInfo): String = {
+    var schema = ""
+    if (lambdaAttributeUnits.isEmpty) {
+      schema = "table"
+    } else {
+      for (unit <- lambdaAttributeUnits) {
+        schema += s"""\"${unit.attributeName}\":${unit.expression},"""
+      }
+      schema = "{" + schema + "}"
+    }
+    s"""
+from pytexera import *
+class ProcessTableOperator(UDFTableOperator):
+    @overrides
+    def process_table(self, table: Table, port: int) -> Iterator[Optional[TableLike]]:
+        yield $schema
```

Comment:
Please rename the variable `schema`. it is confusing. I assume it is a tuple/tuplelike instead?

Reply:
Renamed to `outputTuples`

Reply:
sorry, but I think it should be `outputTable` instead. My previous comment mistakenly indicated it to be tuples. The output should be a TableLike instead.

Reply:
Renamed.